raddar/api/api_v1/endpoints/webhooks.py

```diff
@@ -0,0 +1,43 @@
+from typing import Any, List
+
+from fastapi import APIRouter, Body, Depends, Header, Request
+from sqlalchemy.orm import Session
+
+from raddar.core import contexts, dependencies
+from raddar.crud import crud
+from raddar.schemas import schemas
+from raddar.core.settings import settings
+from raddar.api import deps
+from raddar.lib.managers.detect_secrets_manager import get_project_secrets
+from raddar.lib.managers.repository_manager import get_branch_name
+
+
+router = APIRouter()
+
+
+@router.post("/github", dependencies=[Depends(dependencies.valid_github_webhook)])
+def scan_github_project(payload: dict, db: Session = Depends(deps.get_db)):
```

Comment:
In further iterations we could type this `dict` better, maybe with a Pydantic class that documents which properties we *need* (because the payload comes from a GitHub webhook, we don't control it but we could document what we use from this payload).